src/window_manager.rs

```diff
@@ -14,7 +14,7 @@ trait NodeExt {
 
 impl NodeExt for Node {
     fn is_workspace(&self) -> bool {
-        self.node_type == NodeType::Workspace
+        self.name.as_deref() != Some("__i3_scratch") && self.node_type == NodeType::Workspace
```

Comment:
Could you add a comment here explaining what this `i3_scratch` is? (or in the commit message if you prefer)

Reply:
To be honest IDK what this is but `rename_workspace()` errors when we try to rename it. We just weren't catching these errors previously.

Reply:
I'll add a comment explaining this

Reply:
And is that on i3 only?

Reply:
I use `sway`

Reply:
Some context from looking around a little bit:
https://github.com/swaywm/sway/blob/b8995ced8fb0a4c48300190dfd610f7420aca795/sway/ipc-json.c#L370
https://www.rockyourcode.com/til-about-the-i3-scratchpad/

Reply:
Thanks for pointing out!